### DISCUSSION TITLE
Make the file name more visible in the CMS section

### DISCUSSION THREAD

**Discussion by HitArrowLegend** [wintercms/winter#89] [Ideas]
I was working on my theme and noticed that all the tabs had the same name due to my variables being named as follow : something-variables.scss

I thought it would be nice to either be able to change where you want the cutaway to happen (either at the beginning or at the end) or possibly to have the name goes from right to left, so that we can, at some point, see which file is where.

https://puu.sh/HzrpD/317b79e678.png

**Comment by LukeTowers** [comment]
At a certain number of tabs open at once this is unavoidable regardless of direction of the cutaway. Do the tabs not show the full name on hover?

**Reply by bennothommo** [reply]
The full name does appear on hover.

**Reply by HitArrowLegend** [reply]
Actually the first time I notice it does appear on hover...
I still think it would be nice to be able to choose which part was displayed. Could be a niche demande tho (and it is clearly not urgent)

**Reply by bennothommo** [reply]
In terms of usability, I feel the tabs are good as they are. We want to show as many tabs as possible with as much context as possible.

**Reply by HitArrowLegend** [reply]
I don't ask for that to change, I meant for when the filename is too long. Instead of having only the end being displayed, a user could decide which end would be displayed. In my screenshot that would mean :
/ 01-Color-Variab... \ / 02-Typography-v... \/ 03-Layout-Varia... \ and so on
